spec/02-integration/05-proxy/18-client_certificate_spec.lua

```diff
@@ -0,0 +1,135 @@
+local helpers = require "spec.helpers"
+local ssl_fixtures = require "spec.fixtures.ssl"
+
+
+local fixtures = {
+  http_mock = {
+    upstream_mtls = [[
+      server {
+          server_name example.com;
+          listen 16798 ssl;
```

Comment:
If possible, it'd be better to use a unix socket here, but not a blocker.

Reply:
I don't think we create Services with unix domain sockets as upstream in the test suite so most likely it is not supported.